pyproject.toml

```diff
@@ -42,7 +42,7 @@ requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pytest.ini_options]
-addopts = "--junitxml=test-results/junit.xml --cov --cov-report=html"
+#addopts = "--junitxml=test-results/junit.xml --cov --cov-report=html"
```

Comment:
is it on purpose?

Reply:
Nope, my bad. This was to enable compilation on Pycharm. thanks!